api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java

```diff
@@ -355,7 +351,7 @@ public void createCohortReview() throws Exception {
             + "\"ICD10CM\",\"group\":true,\"attributes\":[],\"ancestorData\":false,\"standard\":false,\"conceptId\":1,\"value\":\"C34\"}],"
             + "\"modifiers\":[]}],\"temporal\":false}],\"excludes\":[]}";
     cohortWithoutReview.setCriteria(criteria);
-    cohortDao.save(cohortWithoutReview);
+    cohortWithoutReview = cohortDao.save(cohortWithoutReview);
 
     org.pmiops.workbench.model.CohortReview cohortReview =
```

Comment:
Can we remove this fully qualified name now that we have DbCohortReview?

Reply:
👍